helm/coco-kafka-bridge/app-configs/kafka-bridge_delivery_prod.yaml

```diff
@@ -48,3 +48,34 @@ bridges:
   type: "proxy"
   replicas: 1
 ## END prod bridges
+
+## Begin COCO prod bridges - TO BE REMOVED BEFORE COCO DECOMM
```

Comment:
Shouldn't we bridge also from coco US here? In case the cluster fails over for some reason?